Doc/library/stdtypes.rst

```diff
@@ -4106,7 +4106,8 @@ its contents cannot be altered after it is created; it can therefore be used as
 a dictionary key or as an element of another set.
 
 Non-empty sets (not frozensets) can be created by placing a comma-separated list
-of elements within braces, for example: ``{'jack', 'sjoerd'}``, in addition to the
+of elements within braces, for example: ``{'jack', 'sjoerd'}``, by using a set
+comprehension, for example ``{x for x in iterable}``, in addition to the
```

Comment:
This should be two separate sentences.  Also, set comprehensions can make both empty and non-empty sets.

Reply:
Thank you for your review. Can you review the changes again?